pkg/cli/zip.go

```diff
@@ -291,7 +291,7 @@ func runDebugZip(cmd *cobra.Command, args []string) (retErr error) {
 
 			zr.sqlOutputFilenameExtension = computeSQLOutputFilenameExtension(sqlExecCtx.TableDisplayFormat)
 
-			sqlConn, err := makeTenantSQLClient(ctx, "cockroach zip", useSystemDb, tenant.TenantName)
+			sqlConn, err := makeTenantSQLClient(ctx, catconstants.InternalAppNamePrefix+" cockroach zip", useSystemDb, tenant.TenantName)
```

Comment:
If you look in `makeSQLClientForBaseURL`: https://github.com/cockroachdb/cockroach/blob/360c7accb958d4a2f79bbf86f11feccbb5154d33/pkg/cli/sql_client.go#L140 

You can see it adds a `$` prefix. Does this mean the client being made in debug zip will be `$$internal cockroach zip`?

Reply:
🤦 Idk why i didn't see the update to sql_client.go. You can ignore this